lms/djangoapps/courseware/courses.py

```diff
@@ -415,3 +416,32 @@ def get_studio_url(course, page):
     if is_studio_course and is_mongo_course:
         studio_link = get_cms_course_link(course, page)
     return studio_link
+
+
+def get_entrance_exam_modules(request, course, student):
+    """
+    This returns all problems in an entrance exam
+    """
+
+    exam_modules = []
+    if settings.FEATURES.get('ENTRANCE_EXAMS', False):
+        if getattr(course, 'entrance_exam_enabled', False):
```

Comment:
If entrance_exam_enable is an (xBlock) field on the course descriptor, this getattr is unnecessary, this can be slightly more legible as:

if course.entrance_exam_enabled: